cgmes/cgmes-conversion/src/test/resources/cim14/tx-from-microBE-adapted.xiidm

```diff
@@ -0,0 +1,56 @@
+                <iidm:step r="-4.7423978591918825" x="-4.7423978591918825" g="4.978498043843005" b="4.978498043843005" rho="0.9760000109672546"/>
+                <iidm:step r="-3.1743946510314203" x="-3.1743946510314203" g="3.2784661036619367" b="3.2784661036619367" rho="0.984000027179718"/>
+                <iidm:step r="-1.593603216552708" x="-1.593603216552708" g="1.6194101894204893" b="1.6194101894204893" rho="0.9919999837875366"/>
+                <iidm:step r="0.0" x="0.0" g="0.0" b="0.0" rho="1.0"/>
+                <iidm:step r="1.6064032684326435" x="1.6064032684326435" g="-1.581005937380453" b="-1.581005937380453" rho="1.0080000162124634"/>
+                <iidm:step r="3.2256065887452223" x="3.2256065887452223" g="-3.124812433019808" b="-3.124812433019808" rho="1.0160000324249268"/>
+                <iidm:step r="4.857609960937737" x="4.857609960937737" g="-4.632577418794237" b="-4.632577418794237" rho="1.0240000486373901"/>
+                <iidm:step r="6.502388780212698" x="6.502388780212698" g="-6.105392427987299" b="-6.105392427987299" rho="1.031999945640564"/>
+                <iidm:step r="8.159992065429833" x="8.159992065429833" g="-7.544371915720527" b="-7.544371915720527" rho="1.0399999618530273"/>
+                <iidm:step r="9.830395402526904" x="9.830395402526904" g="-8.95052354723721" b="-8.95052354723721" rho="1.0479999780654907"/>
+                <iidm:step r="11.51359879150391" x="11.51359879150391" g="-10.324838330283637" b="-10.324838330283637" rho="1.055999994277954"/>
+                <iidm:step r="13.20960223236085" x="13.20960223236085" g="-11.668270157197757" b="-11.668270157197757" rho="1.0640000104904175"/>
+                <iidm:step r="14.918405725097728" x="14.918405725097728" g="-12.981737460563814" b="-12.981737460563814" rho="1.0720000267028809"/>
+                <iidm:step r="16.64000926971454" x="16.64000926971454" g="-14.266124783338041" b="-14.266124783338041" rho="1.0800000429153442"/>
+                <iidm:step r="18.374412866211287" x="18.374412866211287" g="-15.522284268457875" b="-15.522284268457875" rho="1.0880000591278076"/>
+                <iidm:step r="20.121590383911325" x="20.121590383911325" g="-16.751018963037588" b="-16.751018963037588" rho="1.0959999561309814"/>
+                <iidm:step r="21.881593893432694" x="21.881593893432694" g="-17.953156989860908" b="-17.953156989860908" rho="1.1039999723434448"/>
+                <iidm:step r="23.654397454833997" x="23.654397454833997" g="-19.129442981171785" b="-19.129442981171785" rho="1.1119999885559082"/>
+                <iidm:step r="25.440001068115237" x="25.440001068115237" g="-20.2806129237045" b="-20.2806129237045" rho="1.1200000047683716"/>
+                <iidm:step r="27.23840473327641" x="27.23840473327641" g="-21.40737679820408" b="-21.40737679820408" rho="1.128000020980835"/>
+            </iidm:ratioTapChanger>
+        </iidm:twoWindingsTransformer>
+    </iidm:substation>
+</iidm:network>
```

Comment:
add an empty line

Reply:
`.xiidm` files have been created using IIDM `XMLExporter`, and the do not end with a newline character (it happens the same in file `iidm/iidm-xml-converter/src/test/resources/terminalRef.xiidm` for example). 

do you want me to manually add a line ending to these files? should we fix the `XMLExporter`?

Reply:
It is generally good practice to add an extra empty line at the end of the file but if the `XMLExporter` doesn't do it, you can leave as it is, I did not notice this was its default behaviour.